gax/repositories.bzl

```diff
@@ -18,15 +18,15 @@ def com_google_gapic_generator_cpp_gax_repositories():
     _maybe(
         http_archive,
         name = "com_github_grpc_grpc",
-        strip_prefix = "grpc-cc75d93818410e2b0edd0fa3009a6def9ac403ca",
-        urls = ["https://github.com/grpc/grpc/archive/cc75d93818410e2b0edd0fa3009a6def9ac403ca.tar.gz"],
+        urls = ["https://github.com/grpc/grpc/archive/v1.21.0.tar.gz"],
```

Comment:
Nit: can we consistently order the two fields `urls` and `strip_prefix`?

Reply:
reordered (alphabetically)

Reply:
This isn't reordered yet? Shouldn't it be strip_prefix and then urls, like on 28?